actor/echo/README.md

```diff
@@ -1,15 +1,23 @@
 # {{project-name}}
 
-This actor echoes messages received in an HttpRequest.
-It is linked to the http server provider wasmcloud:httpserver.
+This actor echoes messages received in an `HttpRequest`.
+It is linked to the http server provider `wasmcloud:httpserver`.
 
 For each http request, the actor returns a json-formatted
 string containing fields from the request.
 
+## How do I customize this template to use other contracts & interfaces?
+
+- You can change what contracts this actor claims in `wasmcloud.toml` and the `Makefile`. In the future this will just be in `wasmcloud.toml`.
+- You will then need to change the dependencies in `Cargo.toml` to import the interfaces for the contracts you want. Optionally delete the `wasmcloud-interface-httpserver` dep if you're not using that contract.
```

Comment:
change "Optionally delete" to "Delete".
The list of contracts should match exactly what the actor needs to use.
(apply change to all files)

Reply:
Ohh I see what you mean - the "Optionally" and the "if you're not using that contract" imply that its optional to delete the dependency even in the case where you aren't using the contract.  Will fix.